MAINTAINERS.yml

```diff
@@ -2859,17 +2859,21 @@ RISCV arch:
     - carlocaione
     - npitre
     - ycsin
+    - pfarwsi
+    - kanakshilledar
   files:
     - arch/riscv/
     - boards/qemu/riscv*/
     - boards/sifive/
     - boards/sparkfun/red_v_things_plus/
+    - boards/starfive/
```

Comment:
should we introduce a new entry for `Starfive Platforms`?

Reply:
We are planning to add it under the RISCV Arch area.

Reply:
@ycsin Do you think we should create a new entry for Starfive or keep it as it is?